src/Nav.tsx

```diff
@@ -46,6 +69,77 @@ export default function Nav({
   const itemTextStyle = { lineHeight: `${iconWidth}px`, margin: "0px" };
   const drawerWidth = isNavOpen ? 240 : iconWidth + 26;
 
+  function routeDataToIconLinks(routeDatas: RouteData[]): IIconLinkData[] {
+    return routeDatas.map((data) => {
+      let icon = undefined;
```

Comment:
"let" in javascript land is code small. The if else could be refactored to a single statement using a ternary with a const instead of a let.